terraform/application/config/review.tfvars.json

```diff
@@ -2,5 +2,7 @@
   "cluster": "test",
   "namespace": "bat-qa",
   "deploy_azure_backing_services": false,
-  "enable_postgres_ssl": false
+  "enable_postgres_ssl": false,
+  "create_storage_account": true,
+  "storage_account_name": "s189t01ittmsrvsa"
```

Comment:
using the same name for review apps must it will already exist, so can't be created on later rv apps. You'll need to use a local var with the pr number in the name e.g. https://github.com/DFE-Digital/access-your-teaching-qualifications/blob/main/terraform/application/storage.tf#L6

Reply:
I've removed the json configuration for the storage account and added in some code to compute what the account name should be for each environment.